resolwe_bio/expression_filters/sample.py

```diff
@@ -21,7 +21,14 @@ def get_sample_attr(data, attr):
     if not sample_qs.exists():
         return None
 
-    return getattr(sample_qs.first(), attr, None)
+    return sample_qs.first()
+
+
+def get_sample_attr(data, attr):
+    """Get ``attr`` attribute of sample object."""
+    sample = get_sample(data)  # pylint: disable=redefined-outer-name
```

Comment:
Remove pylint disable. You can remove `sample` function and map `'sample'` to `get_sample` in `filters` dict.